bitrise.yml

```diff
@@ -836,11 +836,11 @@ workflows:
 
             ./firefox-ios/l10n-screenshots.sh en-US
         title: Generate screenshots
-    - deploy-to-bitrise-io@1.10:
+    - deploy-to-bitrise-io@2.9.2:
```

Comment:
@clarmso let's see if with this change the L10nBuild passes 🤞

Reply:
Yes.
https://app.bitrise.io/build/243b6dee-1ec1-4c93-9ca2-9b750823901d